### DISCUSSION TITLE
Standup (Monday, November 28th)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [BBGTC/food-bank-app#79] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on profile views - main, donations, personal info (not connected to back)

**What are your priorities for the next 24 hours?**:
 - Finish receeipts view

**Blockers**:
 - Finally recovering from flu

**Shoutouts**:
 - @CarlosCRG19

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Implement Signup navigation logic

**What are your priorities for the next 24 hours?**:
 - Finish that

**Blockers**:
 - Elective stuff for today

**Shoutouts**:
 - @DavBot02 for reviving

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Sent #76 
 - Sent #74 
 - Started 4th deliverable (not as simple as I thought it would be)

**What are your priorities for the next 24 hours?**:
 - Continue with 4th deliverable
 - Finish Chaires'work

**Blockers**:
 - None

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Submitted #77 

**What are your priorities for the next 24 hours?**:
 -  Implement  donation validation

**Blockers**:
 - Lots of homework

**Shoutouts**:
 - @DavBot02  for surviving the gay22 flu
